module1/homework/removeVowels/vowel.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+#include <string>
+#include <vector>
```

Comment:
Header includes and standard library includes should be in the another paragraphs